WasatchNET/SPISpectrometer.cs

```diff
@@ -753,7 +753,7 @@ public override short detectorOffset
         public override short detectorOffsetOdd { get => 0; set { } }
 
         public override bool isARM => false;
-
+        public override bool isInGaAs => false;

```

Comment:
Okay for now, but note that we did provide a 1064 unit to a customer who typically uses SPI comms.

Reply:
I was not aware of that...could you let me know who offline?